utilities/pipelines/deploymentRemoval/Clear-ManagementGroupDeployment.ps1

```diff
@@ -31,13 +34,17 @@ function Clear-ManagementGroupDeployment {
         [string] $ManagementGroupId,
 
         [Parameter(Mandatory = $false)]
-        [string[]] $DeploymentStatusToExclude = @('running', 'failed')
+        [string[]] $DeploymentStatusToExclude = @('running', 'failed'),
+
+        [Parameter(Mandatory = $false)]
+        [int] $maxDeploymentRetentionInDays = 7
```

Comment:
I'd suggest agreeing on the same default value for both the scripts and the pipeline

Reply:
Don't see a reason for it, but ok. :D

Reply:
Updated